apis/nucleus/src/components/listbox/ListBox.jsx

```diff
@@ -64,6 +64,17 @@ function getSizeInfo({ isVertical, checkboxes, dense, height }) {
   };
 }
 
+function SearchMessage(width) {
+  const styleSearchMsg = {
+    fontSize: '16px',
+    padding: '16px 0',
+    textAlign: 'center',
+    minWidth: width,
+  };
+
+  return <div style={styleSearchMsg}>There are no matches for your search.</div>;
+}
+
```

Comment:
Move out in to a new file (`SearchDisclaimer.jsx` or something) and import it from ListBox.jsx. It will make the upcoming merge conflict easier to solve 😄